### ISSUE TITLE
Slashes at the end of locales should be redirected away.

### ISSUE THREAD

**Issue by GoogleCodeExporter** [chrsmithdemos/html5rocks#870]
```
http://www.html5rocks.com/pt --> http://www.html5rocks.com/pt/
```

Original issue reported on code.google.com by `mkwst@google.com` on 18 Apr 2012 at 6:21

**Comment by GoogleCodeExporter** [comment]
```
All yours, Renato!
```

Original comment by `mkwst@google.com` on 20 Apr 2012 at 6:18

**Comment by GoogleCodeExporter** [comment]
```
This issue was closed by revision 579d556cbef6.
```

Original comment by `mang...@html5rocks.com` on 20 Apr 2012 at 6:51
- Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
```
Hey Renato! Thanks for fixing this!

Would you mind making a small change? I'd like to see `/pt` actually redirect 
to `/pt/`. Having two URLs point to the same resource isn't really clean. We 
should have only one URL per resource, if at all possible.

Thanks!
```

Original comment by `mkwst@google.com` on 23 Apr 2012 at 8:38
- Changed state: **Assigned**

**Comment by GoogleCodeExporter** [comment]
```
[deleted comment]
```

**Comment by GoogleCodeExporter** [comment]
```
Yep, absolutely. Just fixed it.

Thanks for the tip! 
```

Original comment by `mang...@html5rocks.com` on 23 Apr 2012 at 8:23

**Comment by GoogleCodeExporter** [comment]
```
This issue was closed by revision 74316b6c7834.
```

Original comment by `mang...@html5rocks.com` on 23 Apr 2012 at 8:25
- Changed state: **Fixed**
